migrations/mysql/20240823165328_modify_access_token_uniq.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE access_tokens
+    ADD revoked_at_is_null BOOLEAN GENERATED ALWAYS AS (revoked_at IS NULL);
+
+CREATE OR REPLACE UNIQUE INDEX idx_account_token_name
+    ON access_tokens (account_id, token_name, revoked_at_is_null);
```

Comment:
1) To fix the previously missed test
2) MariaDB does not support indexes with predicates